tfjs-backend-webgpu/src/matmul_packed_webgpu.ts

```diff
@@ -272,8 +270,9 @@ const readDataFromSubASnippet = (transposeA: boolean) => {
 
 export function makeMatMulPackedSource(
     workPerThread: number[], workGroupSize: [number, number, number],
-    transposeA = false, tileInner = 32, splitK = false,
-    splitedDimInner = 32): string {
+    transposeA = false, tileInner = 32, splitK = false, splitedDimInner = 32,
+    sequentialAccess =
+        false /**By default, it's interval access in threads */): string {
```

Comment:
It's better to move this comment to the earliest place we set this.

Reply:
This is the place that we introduce the `sequentialAccess` concept. So I think it's better to put the comments here. Also updated it with more details.